kernel/kernel-api/src/test/scala/io/delta/kernel/internal/SnapshotManagerSuite.scala

```diff
@@ -524,15 +522,8 @@ class SnapshotManagerSuite extends AnyFunSuite with MockFileSystemClientUtils {
   }
 
   test("getLogSegmentForVersion: versionToLoad not constructable from history") {
-    val files = deltaFileStatuses(20L until 25L) ++ singularCheckpointFileStatuses(Seq(20L))
     testExpectedError[RuntimeException](
-      files,
-      versionToLoad = Optional.of(15),
-      expectedErrorMessageContains = "Cannot load table version 15"
-    )
-    testExpectedError[RuntimeException](
-      files,
-      startCheckpoint = Optional.of(20),
```

Comment:
this just throws an IllegalArgumentException when we invoke the log listing util ... which is expected.